guides/common/modules/proc_configuring-ipxe-to-reduce-provisioning-times.adoc

```diff
@@ -59,7 +59,7 @@ ifdef::foreman-el,katello,satellite,orcharhino[]
 +
 [options="nowrap" subs="+quotes,attributes"]
 ----
-# {package-install} ipxe-bootimgs
+# {client-package-install} ipxe-bootimgs
```

Comment:
This is on a server, so technically this should be 
```suggestion
# {project-package-install} ipxe-bootimgs
```

Because they are the same for all non-satellite, it doesn't matter but it happens on the server.